larq_compute_engine/core/tests/packbits_aarch64_tests.cc

```diff
@@ -15,17 +15,10 @@ using namespace compute_engine::core;
 template <void (*packing_func)(const float*, std::size_t, std::uint64_t*),
           std::size_t blocksize>
 void test_bitpacking() {
-  constexpr std::size_t n = 512;
```

Comment:
Lets make this `64 * 15` or so, so that we also test the "`n=3` blocks left" part of the assembly code.